cql3/statements/create_table_statement.cc

```diff
@@ -229,6 +230,16 @@ std::unique_ptr<prepared_statement> create_table_statement::raw_statement::prepa
     }
 
     stmt->_use_compact_storage = _properties.use_compact_storage();
+    if (stmt->_use_compact_storage) {
+        if (!db.get_config().enable_create_table_with_compact_storage()) {
+            throw exceptions::invalid_request_exception("Support for the deprecated feature of 'CREATE TABLE WITH COMPACT STORAGE' is disabled and will eventually be removed in a future version.  To enable, set the 'enable_create_table_with_compact_storage' config option to 'true'.");
+        }
+        auto msg = "CREATE TABLE WITH COMPACT STORAGE is deprecated and will eventually be removed in a future version.";
+        if (this_shard_id() == 0) {
```

Comment:
This if() doesn't look right. Unless I'm misunderstanding something, the statement doesn't run on every shard. So why limit the logging just to shard 0?

Reply:
As far as I saw, `prepare` does run on every shard, so warning on all of them is spamming the log.